src/platform/imx8/include/platform/drivers/interrupt.h

```diff
@@ -44,6 +44,19 @@
 #define PLATFORM_IRQ_HW_NUM	XCHAL_NUM_INTERRUPTS
 #define PLATFORM_IRQ_CHILDREN	64
 
+/* irqstr_get_sof_int() - Get the SOF interrupt for a shared IRQ_STEER
+ * interrupt
```

Comment:
Description of irqster_get_sof_int() should: Convert IRQ_STEER interrupt to SOF logical interrupt